charts/amazon-cloudwatch-observability/templates/linux/cloudwatch-agent-daemonset.yaml

```diff
@@ -111,4 +111,8 @@ spec:
     valueFrom:
       fieldRef:
         fieldPath: metadata.namespace
+  {{- with .Values.tolerations }}
+  tolerations:
+    {{- toYaml . | nindent 8 }}
```

Comment:
Why is the indent 8? I would have expected it to be 4.
Could you run helm template and paste the output please?

Reply:
You're right, I'm going to fix that.